api/samples/countcalls.c

```diff
@@ -51,7 +51,7 @@
 #    define DISPLAY_STRING(msg) dr_printf("%s\n", msg);
```

Comment:
I wonder if we should put all these macros in a shared header or sthg.  OTOH it is nice to have simpler, self-contained samples so maybe just leave it.

Reply:
Do you want DISPLAY_STRING's param to also be enclosed in parens?

Reply:
No, I think it's safe in that position.